src/screens/Customers/components/RecordsTabMenu.tsx

```diff
@@ -24,23 +21,23 @@ const styles = (theme: Theme) =>
     },
   });
 
-interface TabProps {
-  classes: { tabs: string; tab: string; tabPanel: string };
-  defaultTab: string;
+interface RecordsTabMenuProps {
+  classes: { tab: string; tabPanel: string };
   invoices: MeterReadingRecord[];
   payments: PaymentRecord[];
+  defaultTab?: string;
 }
 
-function TabMenu(props: TabProps) {
+function RecordsTabMenu(props: RecordsTabMenuProps) {
   const { classes, invoices, payments, defaultTab } = props;
-  const [value, setValue] = React.useState(defaultTab);
+  const [value, setValue] = React.useState(defaultTab ? defaultTab : "0");
```

Comment:
I'm confused why these tab ids are strings if they're just indices anyways -- could you just make them integers?

Reply:
![image](https://user-images.githubusercontent.com/20329981/115104655-cba6d900-9f0e-11eb-94bf-4f793b258823.png)